hazelcast/src/test/java/com/hazelcast/quorum/cache/CacheQuorumListenerTest.java

```diff
@@ -166,14 +160,15 @@ public void onChange(QuorumEvent quorumEvent) {
             threeNode.put(generateKeyOwnedBy(h1), "bar");
             fail();
         } catch (Exception e) {
+            EmptyStatement.ignore(e);
         }
         try {
             fourNode.put(generateKeyOwnedBy(h1), "bar");
             fail();
         } catch (Exception e) {
+            EmptyStatement.ignore(e);
```

Comment:
IDEA still creates a warning for the empty catch. This always grabs my attention (yellow alert). Renamed the exception though.